tools/postinstall/apply-patches.js

```diff
@@ -98,9 +98,12 @@ searchAndReplace(
 // Workaround for: https://github.com/bazelbuild/rules_nodejs/issues/1208.
 applyPatch(path.join(__dirname, './manifest_externs_hermeticity.patch'));
 
-// Pre-req for https://github.com/angular/angular/pull/36333. Can be removed
-// once @angular/bazel is updated here to include this patch.
-applyPatch(path.join(__dirname, './@angular_bazel_ng_module.patch'));
+try {
+  // Temporary patch pre-req for https://github.com/angular/angular/pull/36333.
+  // Can be removed once @angular/bazel is updated here to include this patch.
+  // try/catch needed for this the material CI tests to work in angular/repo
```

Comment:
Could you elaborate more on the specifics of why the try-catch is necessary?

Reply:
The patch fails to apply in the angular repo for some reason so the try/catch is needed to make https://github.com/angular/angular/pull/36333 green.

@devversion Do you remember the specifics? I've put this in once before to manage the dependency sandwich but can't recall what the underlying mechanics are.

Reply:
Ahh. I think I sorted it out in my head. The patch is applied cleanly in the components repo as it is applied to the last released @angular/bazel. However, in the angular repo. the patch is applied to the head @angular/bazel, which in the PR that change is actually made and where the components repo has to be updated to be green, the patch is applied to an @angular/bazel package that already has the changes in the patch. So it fails to apply and it will take the catch() code path.

Reply:
Yep. Can confirm that this is why we need the `try/catch`.